Trend Micro/trend-micro-apex-one/_meta/smart-descriptions.json

```
@@ -0,0 +1,294 @@
+[
+  {
+    "value": "Attack discovery/detection on {host.name}. Destination IP: {destination.ip}.",
+    "conditions": [
+      {
+        "field": "destination.ip"
+      },
+      {
+        "field": "host.name"
+      }
+    ]
+  },
+  {
+    "value": "Behavior monitoring event detected on {host.name}. Source IP: {source.ip}. Executable involved: {process.executable}. Event type: {cef.event_type}.",
+    "conditions": [
+      {
+        "field": "event.kind",
+        "value": "event"
+      },
+      {
+        "field": "event.type",
+        "value": "change"
+      },
+      {
+        "field": "event.category",
+        "value": "process"
+      },
+      {
+        "field": "host.name"
+      },
+      {
+        "field": "source.ip"
+      },
+      {
+        "field": "process.executable"
+      },
+      {
+        "field": "cef.event_type"
+      }
+    ]
```

I suggest the following improvement
```suggestion
    "value": "Behavior monitoring: The {rule.name} raised on {host.name}",
    "conditions": [
      {
        "field": "event.dataset",
        "value": "Behavior Monitoring"
      },
      {
        "field": "rule.name"
      },
      {
        "field": "host.name"
      }
    ]
```